src/WebJobs.Extensions.CosmosDB/Constants.cs

```diff
@@ -8,6 +8,7 @@ namespace Microsoft.Azure.WebJobs.Extensions.CosmosDB
     public static class Constants
     {
         public const string DefaultConnectionStringName = "CosmosDB";
+        public const string WebJobsCosmosExtensionName = "CosmosDB";
```

Comment:
Feels silly to have two strings with the same value, but didn't want to rename the connection string const